shared/enclave/server/wallets/ledger.ts

```diff
@@ -34,7 +34,7 @@ const Ledger: WalletLib = {
       const res = await app.getAddress(dpath, false, true);
       return {
         publicKey: res.publicKey,
-        chainCode: res.chainCode
+        chainCode: res.chainCode as string // @todo - figure out the cause of this `string | undefined`
```

Comment:
`getAddress` is declared as
```ts
    public getAddress(
      path: string,
      display?: boolean,
      chainCode?: boolean
    ): Promise<{ publicKey: string; address: string; chainCode?: string }>;
```

If you don't want to cast it to a string manually, you can declare `getAddress` as:
```ts
    getAddress(path: string, display?: boolean, chainCode?: false): Promise<{ publicKey: string; address: string }>;
    getAddress(
      path: string,
      display?: boolean,
      chainCode?: true
    ): Promise<{ publicKey: string; address: string; chainCode: string }>;
```